app/controllers/katello/api/v2/content_exports_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Katello
+  class Api::V2::ContentExportsController < Api::V2::ApiController
+    before_action :find_exportable_content_view_version, :only => [:version]
+
+    api :GET, "/content_exports", N_("Show the export history for a content view version")
```

Comment:
In terms of this user-facing aspect of this API (param descriptions..) I think we should scrub the 'content view version' phrasing because that may not necessarily be true - we could be exporting the library. @parthaa would like your opinion here!

Reply:
I guess that makes sesne. We can add it when we add the library export